src/resources/items/ItemForm/CoreForm.tsx

```diff
@@ -37,6 +41,12 @@ const CoreForm = (props: Props): React.ReactElement => {
     setMediaTypes(data)
   }, [data])
 
+  useEffect(() => {
+    if (isSubmitted) {
+      setItemId(undefined)
+    }
+  }, [isSubmitted, isSubmitting])
```

Comment:
hmm, I'm at my limits of `react` here, but...

If the handler doesn't do anything with `isSubmitting`, then is there any point in listening out for it?

Reply:
If it's not there it does not reset the id for the `save / create`.

Reply:
aah, ok.  So, the state of `isSubmitted` may not have changed, but we still need to clear the item. Got it!